app/scenes/Login/index.js

```diff
@@ -73,11 +89,15 @@ function Login({ location }: Props) {
     env.DEPLOYMENT === "hosted" &&
     (config.hostname ? (
       <Back href={env.URL}>
-        <BackIcon color="currentColor" /> Back to home
+        <Trans>
```

Comment:
Can we wrap the `Trans` around just the text on these two? Otherwise we get tags in the translation string unnecessarily